app/models/concerns/fog_extensions/google/server.rb

```diff
@@ -11,7 +11,6 @@ module Server
 
       attribute :network
```

Comment:
Do we want to delete the attribute than?

Reply:
Currently, using this attribute it constructs network_interface i.e primary one for GCE. 
In case of GCE, at least now there is no way to add multiple network interfaces for hosts on foreman side.

In case of `associate_external_ip`, it just add `:access_configs` inside same network interface.
i.e 
network_interfaces = [ { :network => network, :access_configs => [{:name => 'External NAT', :type => 'ONE_TO_ONE_NAT'}]} ]